starboard/tools/testing/test_runner.py

```diff
@@ -40,11 +40,11 @@
 # pylint: disable=consider-using-f-string
 
 _FLAKY_RETRY_LIMIT = 4
-_TOTAL_TESTS_REGEX = re.compile(r"^\[==========\] (.*) tests? from .*"
```

Comment:
Why are you removing these?

Reply:
The output from RDK has a prefix string in front of our output. 
For example:
Sep 11 23:23:36.071628 WPEFramework[20227]: [  PASSED  ] 2664 tests.
Sep 11 23:23:36.071628 WPEFramework[20227]: [  FAILED  ] 75 tests, listed